packages/ignoreme

```diff
@@ -1 +0,0 @@
-placeholder to create nuget packages folder.
```

Comment:
Was this delete on purpose?

Reply:
it's accidental - I keep doing it ! It's is when I am testing the nuget buids - I'll sort it. I thought I had :-)